core/src/main/java/de/danoeh/antennapod/core/storage/AutomaticDownloadAlgorithm.java

```diff
@@ -52,7 +53,10 @@ public Runnable autoDownloadUndownloadedItems(final Context context) {
 
                 List<FeedItem> candidates;
                 final List<FeedItem> queue = DBReader.getQueue();
-                final List<FeedItem> newItems = DBReader.getNewItemsList(0, Integer.MAX_VALUE);
+                final List<FeedItem> newItems = DBReader.getNewItemsList(
+                        0,
+                        Integer.MAX_VALUE,
+                        DateSortType.DESC.name());
```

Comment:
This can save some vertical space :)

```suggestion
                        0, Integer.MAX_VALUE, DateSortType.DESC.name());
```